manticore/utils/event.py

```diff
@@ -1,13 +1,51 @@
 import inspect
 from weakref import WeakSet, WeakKeyDictionary
-from multiprocessing import Manager
+from types import MethodType
 
 # Inspired by:
 # http://code.activestate.com/recipes/577980-improved-signalsslots-implementation-in-python/
 
 class SignalDisconnectedError(RuntimeError):
     pass
 
+
+def forward_signals(dest, source):
+    ''' 
+        Replicate and forward all the signals from source to dest
+    '''
+    #Import all signals from state
+    for signal_name in dir(source):
+        if not signal_name.startswith('will') and \
+            not signal_name.startswith('did') and \
+            not signal_name.startswith('on'):
```

Comment:
This might catch a lot of unintended fields. Isn't this unnecessary with the `isinstance` below?